index.ts

```diff
@@ -108,6 +108,27 @@ async function handleMessage(
   return { ...response, __webext_messenger__ };
 }
 
+async function handleMessage(
+  message: Message,
+  sender: MessengerMeta
+): Promise<MessengerResponse> {
+  if (message.target) {
+    const publicMethod = getContentScriptMethod(message.type);
+    return handleCall(
+      message,
+      sender,
+      publicMethod(message.target, ...message.args)
+    );
+  }
```

Comment:
This is the core of the forwarder at this point. Information about the original sender is not preserved yet.

I think I will turn `MessengerMeta` into an array where each step just prepends its own "sender" information, sort of like a stack trace.